tracer/build/_build/Build.Steps.cs

```diff
@@ -1312,6 +1312,31 @@ void CopyWaf(string architecture, IEnumerable<string> frameworks, AbsolutePath a
             }
         });
 
+    Target UpdateSnapshots => _ => _
+        .Description("Updates verified snapshots files with received ones")
+        .Executes(() =>
+        {
+            var snapshotsDirectory = Path.Combine(TestsDirectory, "snapshots");
+            var directory = new DirectoryInfo(snapshotsDirectory);
+            var files = directory.GetFiles("*.received*");
+
+            var suffixLength = "received".Length;
+            foreach (var file in files)
+            {
+                var source = file.FullName;
+                var fileName = Path.GetFileNameWithoutExtension(source);
+                if (!fileName.EndsWith("received"))
+                {
+                    Logger.Warn($"Skipping file '{source}' as filename did not end with 'received'");
+                    continue;
+                }
+                var trimmedName = fileName.Substring(0, fileName.Length - suffixLength);
+                var dest = Path.Combine(directory.FullName, $"{trimmedName}verified{Path.GetExtension(source)}");
+                Logger.Info(dest);
```

Comment:
```suggestion
```

Reply:
Fixed in ce10f9d0ef64eefa80181d363e3b7b4a5a95b740